website/database.py

```diff
@@ -187,9 +187,12 @@ def forget_user(self, username):
         for Class in self.get_teacher_classes (username, False):
             self.delete_class (Class)
 
-    def all_users(self):
+    def all_users(self, filtering=False):
         """Return all users."""
-        return USERS.scan()
+        #If we have some filtering -> return all possible users, otherwise return last 500
+        if filtering:
+            return USERS.scan()
+        return USERS.scan(limit=500)
```

Comment:
ok to leave this for now but maybe we want to have an option to really see all users? I know that can become expensive, but maybe behind a tickmark or something if we really need it?

Reply:
This can also be achieved by filtering without any restrictions (settings a starting date at 2000 for example). But I agree that it would be nice to have some dedicated option to achieve this. Will take a look to efficiently re-write the filtering, as we also add some additional filtering options in the near future.

Reply:
Yeah I realized I can use that but it is not very clear (but maybe ok, I am the only user)